cdk/kraken/src/deploy.ts

```diff
@@ -64,12 +78,23 @@ export class DeployJob extends CheckoutJob {
           },
         },
         {
-          name: "Deploy",
-          run: dedent`aws eks --region us-east-1 update-kubeconfig --name production --role-arn arn:aws:iam::\${AWS_ACCOUNT_ID}:role/kubectl
+          name: 'Push to kube-manifests repository',
+          run: dedent`cd kube-manifests
+          mkdir -p \${{ github.repository }}
+          cp -r ../k8s/dist/ \${{ github.repository }}
 
           # get repo name from synth step
           RELEASE_NAME=\${{ steps.synth.outputs.RELEASE_NAME }}
 
+          git add \${{ github.repository }}
+          git commit -m "chore(k8s): deploy $RELEASE_NAME"
+          git push`
+        },
+        {
+          name: "Deploy",
+          run: dedent`aws eks --region us-east-1 update-kubeconfig --name production --role-arn arn:aws:iam::\${AWS_ACCOUNT_ID}:role/kubectl
+          # get repo name from synth step
+          RELEASE_NAME=\${{ steps.synth.outputs.RELEASE_NAME }}
           # Deploy
           kubectl apply -f k8s/dist/ -l app.kubernetes.io/component=certificate
```

Comment:
Since the deploy is product specific, it might make more sense to just run `k apply` on the specific product subfolder (e.g. `k8s/dist/penn-courses`) instead of as a whole